ui/src/components/CopyUrlButton.js

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import { CopyToClipboard } from "react-copy-to-clipboard";
+import { PrimaryButton } from "libs/common";
+
+export default class CopyUrlButton extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      buttonText: "Copy URL"
+    };
+    this.handleButtonClick = this.handleButtonClick.bind(this);
+  }
+
+  handleButtonClick() {
+    // Just change the button text for a second
+    this.setState({ buttonText: "Copied!" });
+    setTimeout(() => this.setState({ buttonText: "Copy URL" }), 1000);
+  }
+
+  render() {
+    const { className } = this.props;
```

Comment:
className isn't used, delete 21 and 24

Reply:
I think it used on 24, the styling is passed in through it.